earth_enterprise/src/common/gedbroot/SConscript

```diff
@@ -38,6 +38,11 @@ gedbroot = env.sharedLib('gedbroot',
                                'QtCore'
                                ])
 
+if env['el_ver'] == 'el8':
+  png_ver = 'png16'
+else:
+  png_ver = 'png15'
```

Comment:
So this would cover ubuntu 18.04/16.04 too...?

Reply:
Assuming that it uses png15. We may need a slight adjustment in logic if we go with 16. Something along the lines of `if not env['el_ver'] or env['el_ver'] == 'el8':`.

Reply:
i think it would be cleaner to make `png_ver` an environment variable instead of `el_ver`

Reply:
I agree with that. I thought of that after I had finished up.